lib/src/sep/0099/hotswap.dart

```diff
@@ -34,42 +35,44 @@ class Hotswap {
     //   hotswapServer,
     //   "hotswap server not found in stellar toml of domain " + domain,
     // );
-    final hotswapServer = "https://" + domain + "/boomerang";
+
     return Hotswap(
-      hotswapServer,
+      "https://dev.api.mykobo.co/boomerang",
       httpClient: httpClient,
     );
   }
 
-  Future<HotswapInfoResponse> info() async {
-    Uri serverURI = Util.appendEndpointToUrl(_hotswapServerAddress, 'info');
+  Future<List<HotswapRoute>> info() async {
+    Uri serverURI = Util.appendEndpointToUrl(_serverAddress, 'info');
     var response = await httpClient.get(serverURI);
     final json = jsonDecode(response.body) as Map<String, dynamic>;
 
-    final result = HotswapInfoResponse.fromJson(json);
+    final result = _HotswapInfoResponse.fromJson(json).hotswapRoutes;
+
     return result;
   }
 
-  Future<Transaction> getSignedTransaction(
-    String accountId,
-    KeyPair signer,
-    StellarSDK _sdk,
-    Network _network,
-  ) async {
-    var account = await _sdk.accounts.account(accountId);
+  Future<Transaction> getSignedTransaction({
+    required String accountId,
+    required KeyPair signer,
+    required StellarSDK sdk,
```

Comment:
Can we not add teh stellar sdk adn the network to the constructors?

Reply:
Done.